### ISSUE TITLE
3. Add clear all button in cart and improve css

### ISSUE THREAD

**Issue by IngerMathildeEiklund** [IngerMathildeEiklund/exam-project-1#29]
- [x] As a user, I want to clear all items from my cart with one action

Other:

- [x] Add missing accessibility on the page.
- [x] Improve the CSS on the page.
